rnd/autogpt_server/autogpt_server/util/process.py

```diff
@@ -13,6 +16,10 @@ class AppProcess(ABC):
     process: Optional[Process] = None
     set_start_method("spawn", force=True)
 
+    set_start_method("spawn", force=True)
```

Comment:
duplicate with the line above

Reply:
whoops I'll remove it on a subsequent PR

Reply:
There is also a left-behind import in `app.py`